packages/config/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "clean": "rimraf ./lib",
     "build": "yarn clean && yarn build:ts",
-    "build:ts": "tsc --declaration -p tsconfig.build.json"
+    "build:ts": "tsc --declaration -p tsconfig.build.json && tsc -p tsconfig.esm.json"
```

Comment:
We don't re-output the declarations. In the export maps (if/when added) we will just map to the top-level.